app/abilities/invoice_ability.rb

```diff
@@ -46,7 +47,8 @@ def in_layer(group = subject.group)
   def in_layer_with_receiver
     return in_layer unless subject.receiver
 
-    in_layer && in_layer(subject.receiver.group.layer_group)
+    group = subject.receiver.is_a?(Group) ? subject.receiver : subject.receiver.group
```

Comment:
Ich verstehe die Unterscheidung hier nicht ganz. Unterscheidest du hier, ob subject eine Invoice oder eine InvoiceList ist? Wenn ja, wäre es nicht klarer, die Bedingung darauf zu schreiben?
```suggestion
    group = subject.is_a?(InvoiceList) ? subject.receiver : subject.receiver.group
```

Reply:
Nein hier geht es bewusst um den `receiver` der InvoiceList.
Seit https://github.com/hitobito/hitobito/commit/cff01288d61211fc682d65c07f0ce2bf9874a533 kann eine Gruppe ein `receiver` sein.
Wenn wir hier die `layer_group` überprüfen wollen, müssen wir da je nach `receiver_type` unterschiedlich hin kommen